app/assets/javascripts/oxalis/model.js

```diff
@@ -163,13 +170,16 @@ export class OxalisModel {
     if (annotation != null) {
       // Make flow happy
       const nonNullAnnotation = annotation;
-      tracing = await doWithToken(token =>
-        Request.receiveJSON(
-          `${nonNullAnnotation.dataStore.url}/data/tracings/${nonNullAnnotation.content.typ}/${
+      const annotationType = nonNullAnnotation.content.typ;
```

Comment:
Would it make sense to extract lines 173 to 182 into a function in `admin_rest_api`?